pmg/templates/call_for_comment_stub.html

```diff
@@ -15,14 +15,16 @@
 {% else %}
   {% set title = result.title %}
   {% set date = result.start_date %}
+  {% set enddate = result.end_date %}
+  {% set closed = result.closed %}
```

Comment:
This may be a problem for search results since we don't index end_date or the closed property. Remind me to look into this.